client/src/redux/sagas/intro.sagas.js

```diff
@@ -1,9 +1,39 @@
-import { put, takeLatest, takeLeading } from 'redux-saga/effects';
+import { put, takeLatest, takeLeading, call } from 'redux-saga/effects';
+import { fetchUtil } from 'utils/fetchUtil';
+
+export async function getIntroMessages() {
+  const result = await fetchUtil({
+    url: `/api/intromessages/`,
+    method: 'get'
+  });
+  return result.response;
+}
 
 export function* fetchIntroMessages() {
   yield put({ type: 'INTRO_FETCH_STARTED' });
   try {
-    const messages = JSON.parse(localStorage.getItem('introMessages')) || {};
+    const componentDictionary = yield call(getIntroMessages);
+    // create complete list of IntroMessages for the user with status
+    // for all messages set to unread (true)
+    const messages = {
```

Comment:
I left the hardcoded list to avoid reworking the redux store since IntroMessages are likely changing in the future.  That being said, I'm happy to make changes or draft a Jira ticket to handle the changes later.

Reply:
follow on ticket sounds good. 👍